packages/fxa-content-server/app/scripts/lib/crypto/totp.js

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import base32Decode from 'base32-decode';
```

Comment:
It is probably worthwhile to add some documentation for future readers why this exist. You could also mention the [configuration](https://github.com/mozilla/fxa/blob/master/packages/fxa-auth-server/config/index.js#L1442) in auth-server.